lib/zipkin-tracer/zipkin_logger_tracer.rb

```diff
@@ -1,17 +1,24 @@
 require 'zipkin-tracer/zipkin_tracer_base'
 require 'zipkin-tracer/hostname_resolver'
+require 'json'
 
 module Trace
   class ZipkinLoggerTracer < ZipkinTracerBase
+    TRACING_KEY = 'Tracing information'
 
     def initialize(options)
       @logger = options[:logger]
+      @logger_accepts_data = @logger.class.name.split('::').first == 'Lorekeeper'
```

Comment:
well, time to improve lorekeeper then 😉 

it feels wrong to have this ultra specific piece of code referencing a gem many people won't know about, especially without any kind of comment.